sat/cli/bootsys/platform.py

```diff
@@ -29,11 +29,13 @@
 import socket
 import time
 import urllib3.exceptions
+from urllib3.exceptions import MaxRetryError
 from collections import namedtuple
 from multiprocessing import Process
 
 from csm_api_client.k8s import load_kube_api
-from kubernetes.client import BatchV1Api
+from kubernetes.client import BatchV1Api, CoreV1Api, ApiException
```

Comment:
Import of `CoreV1Api` will no longer be needed.

```suggestion
from kubernetes.client import BatchV1Api, ApiException
```